src/main/java/com/google/sps/servlets/UserActionServlet.java

```diff
@@ -14,35 +14,37 @@
 
 package com.google.sps.servlets;
 
-import com.google.appengine.api.users.UserService;
-import com.google.appengine.api.users.UserServiceFactory;
 import com.google.gson.Gson;
-import com.google.gson.JsonObject;
-import com.google.sps.util.UserHelper;
+import com.google.sps.data.User;
 import java.io.IOException;
-import java.io.PrintWriter;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/**
+ * provides get method for adding a recipe ID to one of the three User recipe lists.
+ * request parameters: "destination", "inputID"
+ * "destination" may be one of three user recipe list: "cookbook", "userRecipes", "planner".
```

Comment:
It seems error prone to depend on the request having the exact string you expect, could you use an enum instead?

Reply:
I agree with this but I am not sure how I would enforce it from the user's end. How would I make the enum options available to them from a fetch request?

Reply:
Thanks, I I will go with the second solution for now, but perhaps I'll look into @JsType in a future pr